spec/controllers/photos/destroy_spec.rb

```diff
@@ -0,0 +1,108 @@
+# coding: utf-8
+require 'rails_helper'
+require 'fileutils'
+
+RSpec.describe PhotosController, type: :controller do
+  before do
+    @photo_house = create(:photo_house)
+    @photo_mountains = create(:photo_mountains)
+    @photo_waterfall = create(:photo_waterfall)
+    @photo_way = create(:photo_way)
+
+    FileUtils.cp_r("#{Dir.pwd}/public/backup_photos/test", "#{Dir.pwd}/public")
+  end
+
+  after(:all) do
+    FileUtils.cp_r("#{Dir.pwd}/public/backup_photos/test", "#{Dir.pwd}/public")
+  end
+  
+
+  context "指定IDがデータベースに存在する場合" do
+    # データベースから@photo_wayのデータを削除するリクエスト
+    subject { delete :destroy, params: { id: @photo_way.id } }
+
+
+    it "データベースからデータが1件削除されている。" do
+      expect{ subject }.to change(Photo,:count).by(-1)
+    end
+
+
+    it "指定した画像データがデータベースから削除されている。" do
+      subject
+      expect(Photo.exists?(id: @photo_way.id)).to be false
+    end
+
+
+    it "画像ディレクトリ（データベースに登録されているパス）から画像ファイルが1つ削除されている。" do
+      def photos_count
+        original_path = Dir.pwd
+        Dir.chdir("#{Dir.pwd}/public/test")
+        
+        count = Dir.glob("*.*").count
+
+        Dir.chdir(original_path)
+
+        return count
+      end
+      
+      before_delete_count = photos_count
```

Comment:
```suggestion
      before_delete_count = Dir.glob("#{Dir.pwd}/public/test/*.*").count
```
わざわざディレクトリを移動しなくてもいい気がします